tests/plugins/autocomplete/manual/completingkeycodes.md

```diff
@@ -0,0 +1,18 @@
+@bender-tags: 4.10.0, feature, tp3560
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, basicstyles, autocomplete, textmatch
+
+1. Focus the editor.
+2. Type `@`.
+3. Press `arrow down`.
+3. Press `enter`.
```

Comment:
There are two `3.`s - either use subsequent numbers, or just use markdown magic and go with 

```markdown
1. Focus the editor.
1. Type `@`.
1. Press `arrow down`.
1. Press `enter`.
1. Repeat 1-3 and press `tab`.
```

## Result

1. Focus the editor.
1. Type `@`.
1. Press `arrow down`.
1. Press `enter`.
1. Repeat 1-3 and press `tab`.